backend/uasmartsignage/src/test/java/deti/uas/uasmartsignage/controllerTests/MonitorControllerTest.java

```diff
@@ -31,11 +35,21 @@
 
 @WebMvcTest(MonitorController.class)
 @ActiveProfiles("test")
-class MonitorControllerTest {
+@AutoConfigureMockMvc(addFilters = false)
+public class MonitorControllerTest {
```

Comment:
? why was "public" added to this test if the code analysis always complains about public tests? Am I missing something here?